### PR TITLE
style update messages

### PR DESCRIPTION
**Pull request by keithCuniah**
# Description
Update messages 

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

See #2626 

**Type of change**

(Please delete options that are not relevant. Remember to title the PR according to the type of change)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (change restructuring the codebase without changing functionality)
- [ ] Improvement (change adding some improvement to an existing functionality)
- [ ] Documentation update
- [x] Style

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes. And ideally, reference `tests`)

- [ ] Test A

**Checklist**

- [x] I have merged the original branch into my forked branch
- [ ] I added relevant documentation
- [x] follows the style guidelines of this project
- [ ] I did a self-review of my code
- [x] I made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Datasets to be managed only by users with `admin` role ([#2564]).
 - The list of rules is now accessible while metrics are computed. Closes[#2117](https://github.com/argilla-io/argilla/issues/2117)
 - Style updates for weak labelling and adding feedback toast when delete rules. See [#2626](https://github.com/argilla-io/argilla/pull/2626)
+- Update messages. See [#2648](https://github.com/argilla-io/argilla/pull/2648)
```
Comment (frascuchon) [inline CHANGELOG.md]: This message is not useful. Take in mind what is the purpose of the CHANGELOG.md file: show relevant changes between releases. `Update messages ...` says no much about a relevant change. Even more, we have the relevant change in previous entry: `Style updates for weak...` 
```suggestion
```

File: .pre-commit-config.yaml
Comment (frascuchon) [inline .pre-commit-config.yaml:1]: this file is not relevant to the PR, can you please remove it?

**Review by frascuchon — CHANGES_REQUESTED** [review]
review files and changelog please

File: .pre-commit-config.yaml
Comment (frascuchon) [inline .pre-commit-config.yaml:1]: We cannot remove the file, just the changes introduced here. This file exists, just restore the version from releases/1.6.0 branch, please

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Datasets to be managed only by users with `admin` role ([#2564]).
 - The list of rules is now accessible while metrics are computed. Closes[#2117](https://github.com/argilla-io/argilla/issues/2117)
 - Style updates for weak labelling and adding feedback toast when delete rules. See [#2626](https://github.com/argilla-io/argilla/pull/2626)
+- Update messages for weak labelling and rule management toast. See [#2648](https://github.com/argilla-io/argilla/pull/2648)
```
Comment (tomaarsen) [inline CHANGELOG.md]:
```suggestion
- Style updates for weak labelling and adding feedback toast when delete rules. See [#2626](https://github.com/argilla-io/argilla/pull/2626) and [#2648](https://github.com/argilla-io/argilla/pull/2648)
```
My perspective is that these two items are sufficiently related that they can be "merged" to keep the changelog concise. I'll let @frascuchon make the decision though, he oversees the changelog at the moment.

File: frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue [hunk 3]
```diff
@@ -137,7 +137,7 @@ export default {
         feedbackType: "ERROR",
       },
       messageNotLabels:
-        "To create new rules, you need al least two labels. <br />We highly recommended starting by annotating some records with these labels.",
+        "To create new rules, you need al least two labels. <br />We highly recommend starting by annotating some records with these labels.",
```
Comment (tomaarsen) [inline frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue]:
```suggestion
        "To create new rules, you need at least two labels. <br />We highly recommend starting by annotating some records with these labels.",
```
Small typo here.

File: frontend/components/text-classifier/labeling-rules/RuleEmptyQuery.vue [hunk 4]
```diff
@@ -77,9 +77,9 @@ export default {
         feedbackType: "ERROR",
       },
       messageNotLabels: `To create new rules, you need al least two labels.
```
Comment (tomaarsen) [inline frontend/components/text-classifier/labeling-rules/RuleEmptyQuery.vue]:
```suggestion
      messageNotLabels: `To create new rules, you need at least two labels.
```

**Review by tomaarsen — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (frascuchon) [reply CHANGELOG.md]: Fully agree with @tomaarsen

File: frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue (hunk 3 above)
Comment (nataliaElv) [reply frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue]: good catch! Thanks for spotting that 😄

Comment (tomaarsen) [reply frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue]: 😄

**Review by tomaarsen — APPROVED** [review]
Looks perfect! Thanks

**Review by frascuchon — APPROVED** [review]

**Pull request merged**